config/management/src/lib.rs

```diff
@@ -39,6 +40,8 @@ pub mod constants {
 pub enum Command {
     #[structopt(about = "Submits an Ed25519PublicKey for the association")]
     AssociationKey(crate::key::AssociationKey),
+    #[structopt(about = "Create a waypoint and optionally store place it in a store")]
```

Comment:
Nit: "... optionally store place..."? --> "Create a waypoint and optionally place it into a storage backend"?

Reply:
haha